GOVERNANCE.md

```diff
@@ -0,0 +1,69 @@
+
+## Repositories
+The following code repositories are governed by the Cloud Native Buildpacks community and maintained under the [buildpacks](https://github.com/buildpacks) organization.
+
+* [spec](https://github.com/buildpacks/spec)
+* [lifecycle](https://github.com/buildpacks/lifecycle)
+* [pack](https://github.com/buildpacks/pack)
+* [docs](https://github.com/buildpacks/docs)
+* [rfcs](https://github.com/buildpacks/rfcs)
+* [resources](https://github.com/buildpacks/resources)
+
+## Core Team
+The Core team is responsible for the direction of the project (roadmap), subteam leadership, the spec, and cross-cutting concerns. This includes:
+
+* [spec](https://github.com/buildpacks/spec)
+* [rfcs](https://github.com/buildpacks/rfcs)
+
+Core team members are responsible for voting on RFCs and other changes to the project such as changes in leadership or community roles. No company will have a combined voting block of larger than 50%.
+## Sub-teams
+Sub-teams are responsible for narrower sets of concerns related to specific aspects of the project. Each sub-team will include at least one core team member to help align with the broader roadmap.
+### Roles
+
+#### Maintainers
+Maintainers are in charge of the day to day maintenance of the team's projects. They review
+Reviews PRs, ability to merge.
+Ensures code 	quality
+
+New maintainers must already be contributors, must be nominated by an existing maintainer or core team member, and must be elected by a supermajority of the core team. Likewise, maintainers can be removed by a supermajority of the core team or can resign by notifying one of the maintainers.
+
+#### Contributors
+Contributors are those who make regular contributions to the project (documentation, code reviews, responding to issues, participation in proposal discussions, contributing code, etc.). New contributors may be self-nominated or be nominated by existing contributors, and must be elected by a supermajority of that project’s maintainers.
+
+### Implementation Team
+The Implementation team is responsible for maintaining the components that constitute the Cloud Native Buildpacks reference implementation of the specification. This includes:
+
+* [lifecycle](https://github.com/buildpacks/lifecycle)
+* [libbuildpack](https://github.com/buildpacks/libbuildpack)
+
+### Platform Team
+The Platform team is responsible for maintaining the components that constitute the reference platform. This includes:
+
+* [pack](https://github.com/buildpacks/pack)
+* [Tekton Template](https://github.com/tektoncd/catalog/tree/master/buildpacks)
+
+### Learning Team
+The Learning team is responsible for maintaining the project’s documentation, website, and resources. This includes:
+
+* [docs](https://github.com/buildpacks/docs)
+* [resources](https://github.com/buildpacks/resources)
```

Comment:
`samples` seems to make the most logical sense in the Learning team, but for practical reasons is currently maintained by the Platform team

Reply:
We talked about this in the Leadership meeting, and the learning team today isn't owning the `samples` repo, but we should make changes on the Learning team to account for that instead of moving the project.